Sprint-2/implement/contains.test.js

```diff
@@ -16,20 +16,39 @@ as the object doesn't contains a key of 'c'
 // Given a contains function
 // When passed an object and a property name
 // Then it should return true if the object contains the property, false otherwise
+test("if the object contains the property, return 'true'", () => {
+  expect(contains({ a: 1 }, "a")).toEqual(true);
+});
+test("if the object contains the property, return 'true'", () => {
+  expect(contains({ a: 1, "a,s,3": "op" }, ["a", "s", 3])).toEqual(true);
+});
```

Comment:
I feel like this is slightly surprising behaviour - the object doesn't really contain that array, it contains a stringification of that array. But this works, so I guess it is ok to test for...

I would probably use a different name for the test, though, to make it clear how it's different from the test on line 19.